### DISCUSSION TITLE
[Feature] remove, add column or line

### DISCUSSION THREAD

**Discussion by c0ncentus** [iddan/react-spreadsheet#295] [Ideas]
Hi,
I do it myself but it's really silly thing so i don't show my code.

But it would be great to have some features like excel does
![image](https://user-images.githubusercontent.com/25374826/140494240-e9e62864-2c90-4f53-b86c-d94d3be97afe.png)

1) remove value of the specific column/line
2) remove/copy the specific line/column
3) add to the left/right the line/column

We can do it manually but make sens to make it in the package ...
It would be better to activate or not this features 👍 .

**Comment by iddan** [comment]
I agree @c0ncentus, this is a good idea. PRs are welcome.

**Comment by choijun** [comment]
Hi @iddan 

Are you planning to add this feature ?

Thank you.

**Comment by iddan** [comment]
Hey @choijun, there is some core functionality I'm planning to work on in the near future but PRs are welcome!
